### ISSUE TITLE
Problem when running Krusell-Smith

### ISSUE THREAD

**Issue by kyungkeunkim** [econ-ark/DemARK#53]
I get a warning message when I run a cell below.
```
# Put all agents into the economy
KSEconomy_sim = CobbDouglasMarkovEconomy(agents = MyTypes, **KSEconomyDictionary) 
KSEconomy_sim.AggShkDstn = KSAggShkDstn # Agg shocks are the same as defined earlier

for ThisType in MyTypes:
    ThisType.getEconomyData(KSEconomy_sim) # Makes attributes of the economy, attributes of the agent

KSEconomy_sim.makeAggShkHist() # Make a simulated prehistory of the economy
KSEconomy_sim.solve()          # Solve macro problem by getting a fixed point dynamic r
--------------------------------------------------------------------------------

**** WARNING: could not execute multiThreadCommands in HARK.core.Market.solveAgents()  so using the serial version instead. This will likely be slower. The multiTreadCommands() functions failed with the following error: 
 <class 'AttributeError'> : Can't pickle local object 'CobbDouglasEconomy.update.<locals>.<lambda>'
```

**Comment by mnwhite** [comment]
This happens if dill or joblib is not installed, or if there is some issue
with them.

Multithreading has sometimes broken after an update of dill (unable to
pickle some complicated object, like instances of AgentType), and is then
fixed on the next version.

**Comment by llorracc** [comment]
Fixed by [HARK PR #377](https://github.com/econ-ark/HARK/pull/377)
